docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '2'
```

Comment:
You should update the README of this project to detail using this to run it locally, or run tests, as well as how to run it without this (for current prod deployments / people who hate docker, which on MacOS is a valid emotion to have :P)